commons/src/connections/folders.ts

```diff
@@ -16,10 +16,6 @@ export const fetchFolders = async (query: CommonQuery): Promise<Folder[]> => {
   return await fetch(queryString, getFetchConfig(query))
     .then((response) => handleResponse<MiddlewareResponse<Folder[]>>(response))
     .then((json) => {
-      FolderStore.addFolders({
-        queryKey: JSON.stringify(query),
-        data: json.response,
-      });
```

Comment:
Do we not use these stores? Old way of doing this?

Reply:
I think I see - these are in the connections - we're now leaving these for strictly network fetches. The store population takes places in the respective stores (which invokes this).

Wild that this was happening before.

Reply:
Yeah, looks like some copy/pasting that was unnecessary. This shouldn't be needed at all.